app/services/fact_parser.rb

```diff
@@ -129,6 +129,10 @@ def disks_total
   def kernel_version
   end
 
+  # bios information
```

Comment:
We don't use yardoc too much, but I like being explicit in what the interface is.
```suggestion
  # @return [Hash] bios information
```